### PR TITLE
The ShipElement3D was nowhere to be found

### PR DESCRIPTION
**Pull request by salvalatorre**

### CODE REVIEW

**Review by maceligueta — DISMISSED** [review]

File: applications/DEM_application/custom_problemtype/DEMpack/C-DEMPack/kratos.gid/.directory [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [Dolphin]
-SortOrder=1
 Sorting=2
-Timestamp=2014,5,23,12,39,25
+Timestamp=2018,5,15,19,27,15
+Version=4
 ViewMode=1
```
Comment (maceligueta) [inline applications/DEM_application/custom_problemtype/DEMpack/C-DEMPack/kratos.gid/.directory]: This file should not be pushed

File: applications/DEM_application/custom_elements/ship_element.h [hunk 2]
```diff
@@ -33,11 +33,26 @@ namespace Kratos
...
+        
+        // Engine characteristics        
+        double mEnginePower; //60000000; // 60MW for the Arktika-class icebreaker
+        double mMaxEngineForce; //60000000; // with 20MN the ship almost couldn't make it through the ice
+        double mThresholdVelocity; //1.0; // It was set to 3.0 m/s before, which corresponded to a maximum force of 20MN
+        double mEnginePerformance;
+        
+        // Water drag. Applied to de center of mass. We are assuming the ship is moving in the X direction (in GiD)
+        // Drag constant values in the 3 axes
+        // drag_X = 500000; // Such that the X maximum velocity is 11 m/s, which corresponds to Arktika-class icebreakers
```
Comment (maceligueta) [inline applications/DEM_application/custom_elements/ship_element.h:52]: Is it possible to remove some of these comments?

File: applications/DEM_application/custom_problemtype/DEMpack/C-DEMPack/kratos.gid/.directory (hunk 1 above)
Comment (salvalatorre) [reply applications/DEM_application/custom_problemtype/DEMpack/C-DEMPack/kratos.gid/.directory]: OK!

File: applications/DEM_application/custom_elements/ship_element.h (hunk 2 above)
Comment (salvalatorre) [reply applications/DEM_application/custom_elements/ship_element.h:52]: OK!... I left them with the hope that they would help to understand the code...

**Review by maceligueta — APPROVED** [review]

**Pull request merged**